src/panels/lovelace/cards/hui-history-graph-card.ts

```diff
@@ -0,0 +1,169 @@
+import {
+  html,
+  LitElement,
+  TemplateResult,
+  customElement,
+  property,
+  css,
+  CSSResult,
+  PropertyValues,
+} from "lit-element";
+import { classMap } from "lit-html/directives/class-map";
+
+import "../../../components/ha-card";
+import "../../../components/state-history-charts";
+import "../../../data/ha-state-history-data";
+
+import { getRecentWithCache, CacheConfig } from "../../../data/cached-history";
+import { processConfigEntities } from "../common/process-config-entities";
+import { HomeAssistant } from "../../../types";
+import { HistoryGraphCardConfig } from "./types";
+import { LovelaceCard } from "../types";
+import { EntityConfig } from "../entity-rows/types";
+
+@customElement("hui-history-graph-card")
+export class HuiHistoryGraphCard extends LitElement implements LovelaceCard {
+  public static async getConfigElement() {
+    await import(
+      /* webpackChunkName: "hui-history-graph-card-editor" */ "../editor/config-elements/hui-history-graph-card-editor"
+    );
+    return document.createElement("hui-history-graph-card-editor");
+  }
+
+  public static getStubConfig() {
+    return { entities: [] };
+  }
+
+  @property() public hass?: HomeAssistant;
+  @property() public _stateHistory?: any;
```

Comment:
Why is this public?

Reply:
Ah good catch. I made it public in the beginning to update from chile element but that was dumb and not a Lit way to do things 😉